src/averageAges.js

```diff
@@ -15,11 +15,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const newArray = people.filter(person =>
+    century
+      ? person.sex === 'm' && Math.ceil(person.died / 100) === century
+      : person.sex === 'm'
+  );
+
+  const sum = newArray.reduce((result, man) => result + man.died - man.born, 0);
+
+  return sum / newArray.length;
```

Comment:
DRY - don't repeat yourself
you do this stuff 2 time
let's create a new func witch give us avarageAge

```suggestion
  return getAvarageAge(men)
```